bridges/modules/beefy/Cargo.toml

```diff
@@ -0,0 +1,62 @@
+[package]
+name = "pallet-bridge-beefy"
+version = "0.1.0"
+description = "Module implementing BEEFY on-chain light client used for bridging consensus of substrate-based chains."
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
```

Comment:
I think there were several discussions about license, somewhere we can see:
```
license = "Apache-2.0"
```

but also there is a possibility to use:
```
license.workspace = true
```

Reply:
Good point. Addressed here: https://github.com/paritytech/parity-bridges-common/pull/2910. Changed it to `license.workspace = true`

Reply:
Kept the licenses as before as per this thread: https://github.com/paritytech/parity-bridges-common/pull/2910#discussion_r1546385681